sdk/core/azure-core/src/test/java/com/azure/core/util/CoreUtilsTests.java

```diff
@@ -623,4 +623,9 @@ public void addShutdownHookTimeoutCannotBeZero() {
         assertThrows(IllegalArgumentException.class,
             () -> CoreUtils.addShutdownHookSafely(Executors.newSingleThreadExecutor(), Duration.ZERO));
     }
+
+    @Test
+    public void addRuntimeShutdownHookSafelyWithNullThreadDoesNothing() {
```

Comment:
nit: drop `Runtime` from the method name to match the src/main method name.
```suggestion
    public void addShutdownHookSafelyWithNullThreadDoesNothing() {
```